CustomApps/lyrics-plus/Pages.js

```diff
@@ -561,6 +561,13 @@ const GeniusPage = react.memo(
 				ref: c => (container = c),
 				dangerouslySetInnerHTML: {
 					__html: lyrics
+				},
+				onContextMenu: event => {
+					event.preventDefault();
+					const copylyrics = lyrics.replace(/<br>/g, "\n").replace(/<[^>]*>/g, "");
```

Comment:
Wouldn’t this copy the text annotation also?

Reply:
Yes it is

Reply:
That isn’t exactly copying lyrics though, I’d suggest removing text annotations for caching to Unsynced also.

Reply:
Does text annotations mean something like "[verse]"?

Reply:
Text annotations means the explanation that Genius has when you click on a highlighted section of the lyrics

Reply:
If that's the case, I coded not to copy it.

Reply:
It is deleted through the corresponding syntax applied to both files.
```javascript
replace(/<[^>]*>/g, "")
```